### ISSUE TITLE
Not work on M3

### ISSUE THREAD

**Issue by narugit** [narugit/smctemp#21]

**Comment by narugit** [comment]
Need to check below.
- https://github.com/exelban/stats/issues/1703
- https://gist.github.com/tycho/73fb64e7ba0b259923a92a4bacea9e78

**Comment by narugit** [comment]
In above result, it seems `Tp` means performance core and `Te` means efficient core.

**Comment by narugit** [comment]
These performance core value are similar value.
By comparing to M2 sensor https://github.com/narugit/smctemp/blob/8b416626095fc0983ce2421a24a92f1dcaf640ad/smctemp.cc#L435-L450, I will chose 
- Tp01
- Tp09
- Tp0f
- Tp0n
- Tp05
- Tp0D
- Tp0j
- Tp0r

```console
$ smctemp -l | grep "Tp" | grep "4\d\.\d"
  Tp01  [flt ]  45.6 (bytes: B2 39 36 42)
  Tp05  [flt ]  45.6 (bytes: C2 8E 36 42)
  Tp09  [flt ]  45.4 (bytes: EA AB 35 42)
  Tp0D  [flt ]  45.4 (bytes: 10 C6 35 42)
  Tp0H  [flt ]  45.3 (bytes: 0E 57 35 42)
  Tp0L  [flt ]  45.7 (bytes: FA D8 36 42)
  Tp0P  [flt ]  45.3 (bytes: 12 44 35 42)
  Tp0T  [flt ]  45.3 (bytes: C2 36 35 42)
  Tp0X  [flt ]  45.4 (bytes: B2 CA 35 42)
  Tp0b  [flt ]  45.2 (bytes: 48 E7 34 42)
  Tp0f  [flt ]  45.3 (bytes: 06 35 35 42)
  Tp0j  [flt ]  45.6 (bytes: 9C 50 36 42)
  Tp0n  [flt ]  45.3 (bytes: 5E 34 35 42)
  Tp0r  [flt ]  45.4 (bytes: 80 9F 35 42)
  Tp0v  [flt ]  45.1 (bytes: 84 68 34 42)
  Tp0z  [flt ]  45.5 (bytes: D8 00 36 42)
  Tp13  [flt ]  46.2 (bytes: 9C 9D 38 42)
  Tp17  [flt ]  46.4 (bytes: DC 78 39 42)
```

**Comment by narugit** [comment]
Fix by #23
